ghcup-prereleases-0.0.8.yaml

```diff
@@ -251,6 +251,30 @@ ghcupDownloads:
               dlUri: https://downloads.haskell.org/~ghcup/unofficial-bindists/cabal/3.11.0.0.2024.4.19/cabal-install-3.11.0.0-armv7-linux-deb10.tar.xz
               dlHash: 90fed48d921a94bcd0830265d5ccf535a39d3a3e10fc661d9e2cb10798f1a972
               dlOutput: "cabal-install-3.11.0.0.2024.4.19-armv7-linux-deb10.tar.xz"
+    3.15.0.0:
+      viTags:
+        - LatestPrerelease
+      viArch:
+        A_64:
+          Linux_UnknownLinux:
+            unknown_versioning: &cabal-31500-64
+              dlUri: https://github.com/tweag/ghc-wasm-bindists/releases/download/20241019T164140/cabal.tar.xz
```

Comment:
Where are these prereleases cut from?

Reply:
The original URLs are [here](https://github.com/tweag/ghc-wasm-bindists/blob/97bbe50322388e17e957030683b95c5412a8fec6/meta.json#L7-L26). E.g. for x86_64-linux, the CI job is https://gitlab.haskell.org/haskell/cabal/-/jobs/1997392, which is for [this commit](https://gitlab.haskell.org/haskell/cabal/-/commit/96df6f8ef0998f61a205536b2277637a515f7883), a recent `master` commit.

Reply:
"a recent master commit" doesn't seem like it qualifies for naming it `3.15.0.0`. Did you coordinate this versioning with cabal upstream?

Reply:
No, good point. Would changing the version number to sth like `3.15.0.0.2025.10.3` seem acceptable, following the scheme from your commit https://github.com/haskell/ghcup-metadata/commit/eaf801cf9eee107ff2459a653ea5b311a45c27de?

Reply:
Yeah